packages/frontend/src/components/message/Message.tsx

```diff
@@ -475,6 +479,7 @@ export default function Message(props: {
       }
     }
 
+    const Tag = onClick ? 'button' : 'div'
```

Comment:
Maybe "TagName" would be a better name for this? when reading
<Tag className=...> I would expect Tag is a custom component when it just is a string. The same for MessageTag > MessageTagName
Just a proposal...